pkg/cluster/internal/kubeconfig/internal/kubeconfig/paths.go

```diff
@@ -40,7 +40,7 @@ https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
 - If $KUBECONFIG environment variable is set, then it is used as a list of paths (normal path delimiting rules for your system). These paths are merged. When a value is modified, it is modified in the file that defines the stanza. When a value is created, it is created in the first file that exists. - If no files in the chain exist, then it creates the last file in the list.
 
 - Otherwise, ${HOME}/.kube/config is used and no merging takes place.
-*/
+*/ // nolint:lll
```

Comment:
this linter should probably just be disabled. we're not using 80 character-width hardware terminals in 2020.

Reply:
totally align with you there!  on it